### DISCUSSION TITLE
Unrecognized named-value: 'env' error for jobs.<job_id>.container

### DISCUSSION THREAD

**Discussion by ubiuser** [community/community#17311] [Actions]
**Describe the bug**
Usage of `env` in `jobs.<job_id>.container` results in invalid workflow.

**Expected behavior**
`env` should be available as per https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability.

## What's not working?

**Error message**
```
[Invalid workflow file: .github/workflows/integration.yml#L11](https://github.com/99designs/gqlgen/actions/runs/2384254132/workflow)
The workflow is not valid. .github/workflows/integration.yml (Line: 11, Col: 16): Unrecognized named-value: 'env'. Located at position 1 within expression: env.GO_VERSION .github/workflows/integration.yml (Line: 21, Col: 16): Unrecognized named-value: 'env'. Located at position 1 within expression: env.GO_VERSION
```

**Relevant code**
https://github.com/99designs/gqlgen/blob/b262e40a485f67d2659e239a156418938d0fe2e9/.github/workflows/integration.yml

## Job Log Output
https://github.com/99designs/gqlgen/actions/runs/2384254132

**Comment by oleks-popovych** [comment]
subscribed to get an updates

**Comment by germa89** [comment]
I think this is key to foster service containers usage.

**Comment by rhyek** [comment]
This seems to also affect `jobs.<job_id>.environment`.

**Reply by antonblr** [reply]
> This seems to also affect `jobs.<job_id>.environment`.

@rhyek - `env` is not available in `jobs.<job_id>.environment` context - https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability
